### DISCUSSION TITLE
Problemas de Conexión!

### DISCUSSION THREAD

**Discussion by BlazeGroupSAS** [community/community#69301] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

                
2507998 bytes downloaded from github.com at 14.45 Mbps
2507998 bytes downloaded from cloud.githubusercontent.com at 9.06 Mbps
2507998 bytes downloaded from avatars.githubusercontent.com at 30.57 Mbps
2507998 bytes downloaded from github.githubassets.com at 11.58 Mbps
2507998 bytes downloaded from australiaeast.github-debug.com at 2.61 Mbps
2507998 bytes downloaded from brazilsouth.github-debug.com at 21.87 Mbps
2507998 bytes downloaded from centralindia.github-debug.com at 4.19 Mbps
2507998 bytes downloaded from fra.github-debug.com at 5.90 Mbps
2507998 bytes downloaded from iad.github-debug.com at 7.39 Mbps
2507998 bytes downloaded from japaneast.github-debug.com at 4.26 Mbps
2507998 bytes downloaded from koreacentral.github-debug.com at 3.73 Mbps
2507998 bytes downloaded from sea.github-debug.com at 6.79 Mbps
2507998 bytes downloaded from southafricanorth.github-debug.com at 3.99 Mbps
2507998 bytes downloaded from southeastasia.github-debug.com at 3.77 Mbps
2507998 bytes downloaded from github-cloud.s3.amazonaws.com at 8.70 Mbps

                
186.111.141.153

                
Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/117.0.0.0 Mobile Safari/537.36

                
c4578d76331ec51e39acd5b2ed83abc1

                
35FA:DAD2:1F92:24958:651E86FB

                
                

                  
traceroute to 186.111.141.153 (186.111.141.153), 30 hops max, 60 byte packets
 1  *
 2  *
 3  *
 4  *
 5  *
 6  *
 7  *
 8  *
 9  *
10  *
11  *
12  *
13  *
14  *
15  *
16  *
17  *
18  *
19  *
20  *
21  *
22  *
23  *
24  *
25  *
26  *
27  *
28  *
29  *
30  *

                  
PING 186.111.141.153 (186.111.141.153) 56(84) bytes of data.

--- 186.111.141.153 ping statistics ---
3 packets transmitted, 0 received, 100% packet loss, time 2028ms

                  
Start: 2023-10-05T02:52:00-0700
HOST: 5f78d7bc2742dda9f2b5e98d76637098              Loss%   Snt   Last   Avg  Best  Wrst StDev

                  
153.141.111.186.in-addr.arpa domain name pointer host153.186-111-141.telecom.net.ar.

**Comment by ghostinhershell** [comment]
Hey there @BlazeGroupSAS, 

Thanks so much for reaching out here. Are you still having issues accessing Github.com?
